app/client/templates/views/send.html

```diff
@@ -1,7 +1,5 @@
 <template name="views_send">
-
-    {{reactiveData}}
-
```

Comment:
Why is this removed??
This was necessary to trigger reactivity in the respective function